uwblueprint/feeding-canadian-kids
# Users settings UI

## Notion Ticket Link

[Notion Ticket URL](https://www.notion.so/uwblueprintexecs/Dev-e3112e78136f49b4b042e9a0d9df9723?p=1a6ecb348acc4ad792bdb1a53c546b10&pm=s)



## Implementation Description
* Create the user settings page UI for desktop and mobile
* Update the header UI with a gear icon to link to the user settings page



## Steps To Test
1. Make sure the UI looks as intended (we did discuss implementing it differently from the designs)
2. Make sure all form fields are validated to be in the correct format
3. Make sure all required fields must be filled in before saving
4. Make sure it works and looks good on mobile devices



## What Should Reviewers Focus On?
* Make sure the page UI looks good on both desktop and mobile.


## Checklist
